docs/getting-started/running.md

```diff
@@ -32,4 +32,11 @@ There are two ways of running the `deploy` command:
    ![Deploy Run Configuration](../img/deploy_config.png)
 
 3. Click on the run button in the top right corner of the IntelliJ window to deploy your robot
-   program to the robot.
\ No newline at end of file
+   program to the robot.
+
+## Simulating Robot Code
+
+FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so,either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" keyword with the "simulateJava" keyword(I.E `./gradlew simulateJava` or `./gradlew.bat simulateJava` on windows).
```

Comment:
`gradlew.bat` should execute as gradlew on windows. Executing scripts behaves differently between cmd and pwsh so we should specify which one is used.
```suggestion
FRC code made with kotlin can be simulated on your computer, just like robot code written with Java, Python, or C++. To do so,either follow the "Deploying via IntelliJ" or "Deploying manually via the command line" instructions above, but replace the "deploy" keyword with the "simulateJava" keyword(I.E `./gradlew simulateJava` on Windows Powershell or a Unix shell).
```